bigquery/tests/unit/test_client.py

```diff
@@ -4636,10 +4636,13 @@ def test_insert_rows_w_schema(self):
         ]
 
         def _row_data(row):
+            result = {"full_name": row[0], "age": str(row[1])}
             joined = row[2]
-            if isinstance(row[2], datetime.datetime):
-                joined = _microseconds_from_datetime(joined) * 1e-6
-            return {"full_name": row[0], "age": str(row[1]), "joined": joined}
+            if joined is not None:
```

Comment:
Nit: I'd prefer less nesting here.

```
if isinstance(joined, datetime.datetime):
    joined = _microseconds_from_datetime(joined) * 1e-6
if joined is not None:
    result["joined"] = joined
```

Reply:
Well, _flat is better then nested_, so I'll do it. :+1: